### ISSUE TITLE
Meeting Agenda December 12, 2023

### ISSUE THREAD

**Issue by Piyopiyo-Kitsune** [WordPress/Learn#2100]
Training Team meetings start at 07:00 UTC weekly on Tuesdays. Anyone is welcome to contribute to the meeting by commenting in Slack threads throughout the week. Meeting notes will be published at the end of the week to also incorporate these asynchronous conversations.

_This week’s meeting will be hosted by @Piyopiyo-Kitsune_

## 1. Intro/Welcome

Welcome to all new teammates who joined the Training Team this week!

## 2. News

**1. [Meeting Note Takers](https://make.wordpress.org/training/handbook/about/team-roles/)**

- December 12 – `@Sumit Singh`
- December 19 –  `@rabin kumar pal`

**Final meeting for the year is next Tuesday, December 19th. We’ll resume on January 9th.**

**2. Looking for feedback**

- item 1
 
**3. Looking for volunteers**

- [Training Team 2023 Year-End Review](https://make.wordpress.org/training/2023/11/20/training-team-2023-year-end-review/)
  - Plan promotions with the Marketing Team
  - Establish monthly recurring onboarding Online Workshops in different timezones for training team roles
  - Figure out a method to manage localized content translation
  - Audit Handbook -- thank you to Abha and Sumit for your assistance here!

**4. Updates from last week's dev-squad triage session**

Attendees: @psykro, @ devmuhib, @kafleg, @bsanevans, @digitalchild, @sumitsingh

PRs triaged:
- #2072, this PR has subsequently been merged and is waiting on a deployment.

Bugs triaged:
- #1772 
- #1659 
- #1782 
- #1860 
- #1875 

Notes: 
- #1772 is an issue to for the Learning Pathways project team to keep in mind, as it relates to using featured images for content.
- #1860, @digitalchild has researched options there and added them in the [comments](https://github.com/WordPress/Learn/issues/1860#issuecomment-1845032745)
- #1875 needs testing, so we're calling on any developers who can help us test that issue, and report back in the comments.
- @bsanevans also started a discussion around how we use the [LearnWP Website Development GitHub project board](https://github.com/orgs/WordPress/projects/71) to manage any issues (and prs) related to the [Learn website](https://learn.wordpress.org/). [You can follow that discussion in Slack](https://wordpress.slack.com/archives/C01KC5VDQBC/p1701935073064809).

**5. Other news**

- [Meet your 2024 Training Team Representatives](https://make.wordpress.org/training/2023/12/07/meet-your-2024-training-team-representatives/) - in 2024 we will welcome Laura Adamonis as a co-representative alongside returning members Benjamin Evans and Destiny Kanno. 
- [Hosting Learn videos on YouTube](https://make.wordpress.org/training/2023/12/06/hosting-learn-videos-on-youtube/) -- this is now adopted by the team! Thank you everyone for your feedback
- [Recap: GitHub updates and process refinements](https://make.wordpress.org/training/2023/12/05/recap-github-updates-and-process-refinements/)
- **Happening this week are two GitHub Updates Online Workshops** -- join @bsanevans for a walkthrough of the GitHub process updates in the WordPress Training team. Two sessions are offered below:
  - [December 14 at 08:00 UTC](https://www.meetup.com/learn-wordpress-online-workshops/events/297747513/) and
  - [December 15 at 00:00 UTC](https://www.meetup.com/learn-wordpress-online-workshops/events/297748992/)
 - [Embracing Matrix for Enhanced Communication](https://make.wordpress.org/project/2023/11/30/embracing-matrix-for-enhanced-communication/) -- this is an important change for how Make teams communicate with each other so please don't skip reading this and testing Matrix out for yourself.
- Handbook pages updated
  - [Scheduling an Online Workshop](https://make.wordpress.org/training/handbook/online-workshops/scheduling-an-online-workshop/)
  - [Editing the Training Team Handbook](https://make.wordpress.org/training/handbook/faculty-program/editing-the-training-team-handbook/)
  - [Expectations for active Faculty members](https://make.wordpress.org/training/handbook/faculty-program/expectations-for-active-faculty-members/)
  - [GitHub Labels](https://make.wordpress.org/training/handbook/training-team-how-to-guides/how-we-use-github/github-labels/)
  - [Automated Workflows](https://make.wordpress.org/training/handbook/training-team-how-to-guides/how-we-use-github/automated-workflows/)
  - [How We Use GitHub](https://make.wordpress.org/training/handbook/training-team-how-to-guides/how-we-use-github/)
- December Monthly Update and January Learn Newsletters will be posted on January 8th, 2024.

## 3. [Open requests for review](https://github.com/orgs/WordPress/projects/33/views/17)

See our [Guidelines for reviewing content](https://make.wordpress.org/training/handbook/training-team-how-to-guides/guidelines-for-reviewing-content-on-learn/) to review the following content.

- **Tutorials**
  - 
- **Lesson Plans**
  - 
- [Translated Content looking for reviewers](https://github.com/orgs/WordPress/projects/104/views/10)
  -
- **Learning Pathway Lessons**
  - 

## 4. Project updates

- [Project Thread: Learning Pathways on Learn WordPress](https://make.wordpress.org/training/2023/07/07/project-thread-learning-pathways-on-learn-wordpress/)

## 5. Open Discussions

If you have topics you’d liked discussed in the meeting, please leave them as a comment on this issue.

---

You can see all meetings scheduled on [this meeting calendar](https://make.wordpress.org/meetings/#training). If you are new to the Training Team, then come walk through [our onboarding program](https://make.wordpress.org/training/handbook/getting-started/) to get to know the team and how we work. And if you have questions, feel free to reach out in [the #training Slack channel](https://wordpress.slack.com/archives/C02RW657Q) at any time.

**Comment by kaitohm** [comment]
Noting @nftushar has volunteered to take notes for the January 23rd meeting next year. ([Slack thread](https://wordpress.slack.com/archives/C02RW657Q/p1702272148426419?thread_ts=1700552689.983839&cid=C02RW657Q))

**Comment by jonathanbossenger** [comment]
I have added the dev-squad triage feedback. 

@Piyopiyo-Kitsune would it be possible to post each of the items under **Notes** as separate messages during the meeting, to allow threadable conversations on each one?

**Comment by ghost** [comment]
The switch to Matrix is on hold as per Matt at State of the Word Q& A section. Still good to continue to learn how to use it.

**Comment by kaitohm** [comment]
This meeting started here: https://wordpress.slack.com/archives/C02RW657Q/p1702364481168269

**Comment by webtechpooja** [comment]
Meeting recap notes: https://make.wordpress.org/training/2023/12/16/training-team-meeting-recap-12th-december-2023/
